### PR TITLE
Upgrade to CakePHP 5.0 for UBI8, UBI9 and PHP 8.1+

### PR DESCRIPTION
**Pull request by phracek**
I did all steps as they were mentioned here: https://github.com/sclorg/cakephp-ex/pull/143

@cuppett  Can you please review it?

### CODE REVIEW

**Review by cuppett — CHANGES_REQUESTED** [review]
We're going to need to add more imagestreams as well. The default in these template BuildConfig files is PHP 7.4:

```yaml
  strategy:
    type: Source
    sourceStrategy:
      from:
        kind: ImageStreamTag
        namespace: openshift
        name: 'php:7.4-ubi8'
      env:
        - name: COMPOSER_MIRROR
```

Here's what's currently available in CRC at 4.14.3:

image

Using this branch (versus the 4.5 one) as the default has a pre-req of getting the 8.1 and 8.2 php image streams registered on a system.

File: openshift/templates/cakephp.json
Comment (cuppett) [inline openshift/templates/cakephp.json:1]: @phracek I don't think that image stream exists, are we having it added somewhere else?

This is what I get making an update in my CRC:

 Error
Error "Error resolving ImageStreamTag php:8.1-ubi9 in namespace openshift: unable to find latest tagged image" for field "from".


Also, the other 2 template files have the same default (cakephp-mysql, cakephp-mysql-persistent), do we want to update those as well?

Comment (cuppett) [reply openshift/templates/cakephp.json:1]: It does exist (in RedHat), I'm just not sure it's part of the streams we lay on the cluster by default today. Here's the references to those locations in another project I've got:

https://github.com/cuppett/ubi-php/blob/main/build/openshift/imagestream.yaml

Comment (phracek) [reply openshift/templates/cakephp.json:1]: The container is here: https://catalog.redhat.com/software/containers/ubi9/php-81/62e8e662f6d3d47e19779b01?q=php-&architecture=amd64&image=657169b9160f22feda11c466&gti-tabs=unauthenticated

Comment (phracek) [reply openshift/templates/cakephp.json:1]: Imagestreams are added here: https://github.com/sclorg/s2i-php-container/pull/432

**Pull request merged**